test/hotspot/jtreg/compiler/exceptions/OptimizeImplicitExceptions.java

```diff
@@ -195,6 +195,16 @@ public static void main(String[] args) throws Exception {
             return;
         }
 
+        // The following options are both develop, getBooleanVMFlag() returns NULL in product build.
+        // If they are set in debug build, disable them for more test stability.
+        if (WB.getBooleanVMFlag("DeoptimizeALot")) {
```

Comment:
If this really returns NULL in a product build, wont this cause an exception then?

Wouldn't it be easier to just unconditionally set them both to false:
```
WB.setBooleanVMFlag("DeoptimizeALot", false);
WB.setBooleanVMFlag("DeoptimizeRandom", false);
```

Reply:
I prefer Volker style rather than @run annotation.   Here are my arguments. 

1. They need an extra one `-XX:+IgnoreUnrecognizedVMOptions` because those 2 options are not valid in debug builds.
2. Current makefile does put JTREG="VM_OPTIONS=-XX:+DeoptimizeALot" before, `@run` options will overwrite them.  We better off not reply on makefile's logic.
3. Writing them in code gives us an opportunity to write a comment.

Reply:
Okay, fine with me.